### PR TITLE
add OLCNE 1.1 support

### PR DESCRIPTION
**Pull request by rosemarymarano**
As per OWLS-OWLS-82400: WKO -Document OLCNE 1.1 certification, added certified support for OLCNE 1.1 with Kubernetes 1.17.0. Modified Release Notes and Prerequisites accordingly. Please note the discrepancy in Kubernetes support (in the RN, Kubernetes 1.16, 1.17, and **1.18** support; in the prerequisites, Kubernetes 1.14.8+, 1.15.7+, 1.1.6.0+, 1.17.0+).

### CODE REVIEW

File: docs-source/content/userguide/introduction/introduction.md [hunk 1]
```diff
@@ -16,10 +16,9 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
 
 ### Operator prerequisites
 
-For the current production release 2.5.0:
+For the current production release 2.6.0:
 
-* Kubernetes 1.13.5+, 1.14.8+, and 1.15.7+ (check with `kubectl version`). Not supported on Kubernetes 1.16 or later; see note below.
-  See note below for OpenShift.
+* Kubernetes 1.14.8+, 1.15.7+, 1.1.6.0+, 1.17.0+ (check with `kubectl version`).
```
Comment (mriccell) [inline docs-source/content/userguide/introduction/introduction.md:21]: Should 1.18.0+ be added?

Comment (rosemarymarano) [reply docs-source/content/userguide/introduction/introduction.md:21]: Yes, if we are supporting it. I'll add it.

Comment (rosemarymarano) [reply docs-source/content/userguide/introduction/introduction.md]: Done.

**Comment by rosemarymarano** [comment]
I updated the operator version in the docs from 2.5.0 to 2.6.0, with this exception: I did not change this statement, "Operator 2.5.0+ is certified for use on OpenShift Container Platform 4.3.0+ with Kubernetes 1.16.2+" at http://localhost:1313/weblogic-kubernetes-operator/userguide/introduction/introduction/

**Pull request merged**